src/Api/KeyManagement/Queries/UserAccountKeysQuery.cs

```diff
@@ -0,0 +1,25 @@
+﻿#nullable enable
+
+using Bit.Core.Entities;
+using Bit.Core.KeyManagement.Models.Data;
+using Bit.Core.KeyManagement.Repositories;
+
+namespace Bit.Api.KeyManagement.Queries;
+
+public interface IUserAccountKeysQuery
+{
+    Task<UserAccountKeysData> Run(User user);
+}
```

Comment:
💭 
Typically, in C# and .NET there is one class per file. This goes for interfaces too, which typically get their own file.

For the most part this is what the current code base does.